app/controllers/api/account.php

```diff
@@ -3587,7 +3593,12 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
                 $abuse = new Abuse($timelimit);
                 if ($abuse->check() && System::getEnv('_APP_OPTIONS_ABUSE', 'enabled') === 'enabled') {
                     $helper = PhoneNumberUtil::getInstance();
-                    $countryCode = $helper->parse($phone)->getCountryCode();
+
+                    try {
+                        $countryCode = $helper->parse($phone)->getCountryCode();
```

Comment:
Also this phone number comes from the user object, when creating a user we use a phone validator as well. We need to fix it there too